cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1646,6 +1649,22 @@ def default_for(
 
                    :cylc:conf:`global.cylc[platforms][<platform name>]hosts`
             ''')
+            Conf('communication method',
+                 VDR.V_STRING, 'zmq',
+                 options=[meth.value for meth in LocalCommsMeth], desc='''
+                The communication method used by the command line interface and
+                local task messaging to interact with running workflow(s).
+
+                Options:
+
+                zmq
+                   Direct client-server TCP communication via network ports
+                https
+                   Via the running Hub proxy and/or UI-Server (requires UI
+                   Server installation)
+
+                .. versionchanged:: 8.2.0
+            ''')
```

Comment:
Can't remember where we got to with this, but I think we can use the `[localhost]` platform to configure the CLI since the config applies the CLI as much as it does job communication (which is over the same CLI).

Reply:
Yeah, this is done

Reply:
There are still two `communication method` sections in the global config though?

Reply:
Yes, still in both sections. I don't want to rule out using `https` for remote jobs.

Reply:
As, sorry, hadn't spotted that this section was just the `localhost` default.